src/components/FileUpload/FileUpload.js

```diff
@@ -1,12 +1,77 @@
-//reference: https://www.positronx.io/react-file-upload-tutorial-with-node-express-and-multer/
-import React, { Component } from 'react';
-import { FormGroup } from '../Utils/Utils';
-export default class FilesUploadComponent extends Component {
-  render() {
-    return (
-            <div className="form-group">
-              <input type="file" name="uploaded_file" onChange={this.props.onChange} />
-            </div>
-    )
-  }
-}
+import React, { useState } from "react";
+import Cropper from "react-cropper";
+import "cropperjs/dist/cropper.css";
+import "./FileUpload.css";
+
+export default function FileUpload(props) {
+  const { handler, imgHandler } = props;
+
+  const [image, setImage] = useState();
+  const [saved, setSaved ] = useState("");
+  const [showCropImage, setCropImage ] = useState("hide");
+  const [cropData, setCropData] = useState("#");
```

Comment:
When running this, I get a warning in React that says 'cropData' is assigned a value but never used. This variable looks important, but I just want to make sure it's doing something

Reply:
Removed unused code. Thanks!